CHANGELOG.md

```diff
@@ -245,6 +245,45 @@ rails decidim:active_storage_migrations:migrate_inline_images_to_active_storage[
 - **decidim-core**: Disable unconfirmed access by default [\#8233](https://github.com/decidim/decidim/pull/8233)
 - **decidim-meetings**: Rename 'upcoming events' content block to 'upcoming meetings' [\#8412](https://github.com/decidim/decidim/pull/8412)
 - **decidim-core**: Change user workflows to prevent user enumeration attacks [\#8537](https://github.com/decidim/decidim/pull/8537)
+### Rename data portability to download your data
```

Comment:
I think we don't want to backport the CHANGELOG changes.

Is that right @andreslucena ?